frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/breakdown-card/utils.tsx

```diff
@@ -19,7 +19,7 @@ const addOthers = (
     name: 'Others',
     color: Colors.OTHER,
     label: humanize(others).string,
-    link: '',
+    link: 'All other capacity usage that are not a part of the top 5 consumers',
```

Comment:
I think we can move this string too to constant.

Reply:
This is only at one place, we can leave it.
If changed it will be here only. WDYT ?

Reply:
Since we are moving all the constant to a single file, I will prefer to keep this there as we know all the constants will be placed there.

Reply:
okay